MovieQuizTests/MovieQuizPresenterTests.swift

```diff
@@ -0,0 +1,31 @@
+import XCTest
+@testable import MovieQuiz
+
+final class MovieQuizViewControllerProtocolMock: UIViewController, MovieQuizViewControllerProtocol {
+    func show(quiz step: QuizStepViewModel) {}
+    
+    func show(quiz result: QuizResultsViewModel) {}
+    
+    func highlightImageBorder(isCorrectAnswer: Bool) {}
+    
+    func showLoadingIndicator() {}
+    
+    func hideLoadingIndicator() {}
+    
+    func showNetworkError(message: String) {}
+}
+
+final class MovieQuizPresenterTests: XCTestCase {
+    func testPresenterConvertModel() throws {
+        let viewControllerMock = MovieQuizViewControllerProtocolMock()
+        let sut = MovieQuizPresenter(viewController: viewControllerMock)
+        
+        let emptyData = Data()
+        let question = QuizQuestion(image: emptyData, text: "Question Text", correctAnswer: true)
+        let viewModel = sut.convert(model: question)
+        
+         XCTAssertNotNil(viewModel.image)
```

Comment:
Control + i чтобы поправить форматриование
```suggestion
        XCTAssertNotNil(viewModel.image)
```